Sources/OpenGraph/Runtime/Metadata.swift

```diff
@@ -7,59 +7,65 @@
 
 public import OpenGraphCxx
 #if canImport(ObjectiveC)
-public import Foundation
+import Foundation
 #endif
 
 @_silgen_name("OGTypeApplyFields")
-public func OGTypeApplyFields(
-    _ type: Any.Type,
+private func OGTypeApplyFields(
+    of type: Any.Type,
     body: (UnsafePointer<Int8>, Int, Any.Type) -> Void
 )
 
 @_silgen_name("OGTypeApplyFields2")
-public func OGTypeApplyFields2(
-    _ type: Any.Type,
-    options: OGTypeApplyOptions,
+private func OGTypeApplyFields2(
+    of type: Any.Type,
+    options: Metadata.ApplyOptions,
     body: (UnsafePointer<Int8>, Int, Any.Type) -> Bool
 ) -> Bool
 
+public func forEachField(
+    of type: Any.Type,
+    do body: (UnsafePointer<Int8>, Int, Any.Type) -> Void
+) {
+    OGTypeApplyFields(of: type, body: body)
+}
+
 extension Metadata: Swift.Hashable, Swift.CustomStringConvertible {
-    @inlinable
-    @inline(__always)
     public init(_ type: any Any.Type) {
         self.init(rawValue: unsafeBitCast(type, to: UnsafePointer<_Metadata>.self))
     }
-    
-    @inlinable
-    @inline(__always)
+
     public var type: any Any.Type {
         unsafeBitCast(rawValue, to: Any.Type.self)
     }
-    
-    @inlinable
-    @inline(__always)
+
     public var description: String {
         #if canImport(ObjectiveC)
         __OGTypeDescription(self) as NSString as String
         #else
         fatalError("Unimplemented")
         #endif
     }
-    
-    @inlinable
-    @inline(__always)
-    /* public */func forEachField(
-        do body: (UnsafePointer<Int8>, Int, Any.Type) -> Void
-    ) {
-        OGTypeApplyFields(type, body: body)
-    }
-    
-    @inlinable
-    @inline(__always)
+
     public func forEachField(
-        options: OGTypeApplyOptions,
+        options: ApplyOptions,
         do body: (UnsafePointer<Int8>, Int, Any.Type) -> Bool
     ) -> Bool {
-        OGTypeApplyFields2(type, options: options, body: body)
+        OGTypeApplyFields2(of: type, options: options, body: body)
+    }
+}
+
+extension Signature: Equatable {
```

Comment:
NOTE: I prefer not add retroactive here since it is not needed. And the warning will be removed in future release of Swift.

Reply:
cc @jcmosc